src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -6,12 +6,10 @@
 
 	public FIFORhymer() {
 		super();
-		// TODO Auto-generated constructor stub
 	}
 
 	public FIFORhymer(IntArrayStack intArrayStack) {
 		super(intArrayStack);
-		// TODO Auto-generated constructor stub
 	}
 
 	public final defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
```

Comment:
2.2.8 still missing. You were to use an IntArrayStackInterface type object instead of a rhymer.